src/core/modules/Asterism.cpp

```diff
@@ -127,16 +207,18 @@ bool Asterism::read(const QJsonObject& data, StarMgr *starMgr)
 			}
 			case Type::TelescopicAsterism:
 			{
+					QMessageBox::information(nullptr, "Bad asterism", "Asterism with coord list found: " + id);
```

Comment:
Is the issue so serious to warrant a message box while other SC issues are simply logged? This may be a dozen times that the user will have to click _OK_ if multiple asterisms have coordinate lists.
Also, this `nullptr` parent can make the message box hidden behind the Stellarium window, making the latter seem frozen.

Reply:
All asterisms in the default SCs are fixed. I wanted to make sure to signal, very loudly and directed to the SC author, that the coordinate lists are no longer supported. Should we hide this notice and just log an "ignore"?

Reply:
We don't do this for any other issue, however serious it may be. And most external SCs are very imperfect format-wise anyway, so their authors will get other errors in the log anyway and should look for them.